cogs/helper.py

```diff
@@ -1,40 +1,89 @@
 import discord
 from discord.ext import commands
+from utils.config import subjects 
 
 
 class Helper(commands.Cog):
     def __init__(self, bot: commands.Bot) -> None:
-        self.bot = bot
-
+        self.bot = bot        
+        self.subjects_list = [pair.split(",") for pair in subjects.split(";")]
+        self.subject_channels_list = [int(pair[0]) for pair in self.subjects_list]
+        
+        @bot.tree.context_menu(name="Toggle Pin")
+        async def pin_message(interaction: discord.Interaction, message: discord.Message):
+            # checks if the command is being issued in a subject channel
+            if interaction.channel.id not in self.subject_channels_list:
+                return await interaction.response.send_message('You may only pin messages in subject channels')
+            # checks if the person issuing the action is a helper
+            subject_helper_role  = [int(list[1]) for list in self.subjects_list if str(interaction.channel.id) in list][0]
+            if subject_helper_role not in [role.id for role in interaction.user.roles]:
+                return await interaction.response.send_message('Only subject helpers can pin/unpin messages')
+            # unpins the message if pinned already
+            if message.pinned:
+                await message.unpin()
+                return await interaction.response.send_message('The message was successfully unpinned')
```

Comment:
If you return here, is the `else` clause necessary?

Reply:
IIRC it exits the function once you return, so there's no point having an else.